src/pages/components/modal/usage.mdx

```diff
@@ -119,6 +122,17 @@ destructive or irreversible then use a transactional danger modal.
 | [Acknowledgment](/components/modal/usage#acknowledgment-modal) | System requires an acknowledgement of the information from the user. Contains only a single button, commonly "OK".              |
 | [Progress](/components/modal/usage#progress-modal)             | Requires several steps to be completed before it can be closed. Contains a cancel, previous, and next/completion buttons.       |
 
+### Feature flags
+
+A
+[feature flag](https://carbondesignsystem.com/components/overview/feature-flag/)
```

Comment:
When creating internal link you don't need to do the whole URL, just an FYI
```suggestion
[feature flag](/components/overview/feature-flag/)
```

Reply:
Oh, is it? That's something new I learned! Thanks, @aagonzales. I’ll update the same in the other feature flags' PRs as well. Thank you!